documentation/docu.adoc

```diff
@@ -0,0 +1,30 @@
+= Strimzi: Kafka as a Service running on Kubernetes and OpenShift
+Version 0.1
+:sectnums:
+:toc: preamble
+:toclevels: 3
+:toc-title: Content
+:imagesdir: ./img
+:xrefstyle: full
+
+This guide describes how to install and use Strimzi. Strimzi provides a way to run an Apache Kafka cluster on Kubernetes
```

Comment:
I'm not native english of course but what about "... and use Strimzi which provides ..." instead of having two separate sentences ?

Reply:
AFAIK English normally prefers several short sentences than one long sentence. But here we might get rid of having twice Strimzi one after another. WDYT @tombentley?

Reply:
Imho the two sentences are really saying two different things. One is saying what Strimzi is, the other saying what this document is about, so two sentences works better. That said, I think we should swap the sentences around, so we have the sentence explaining what Strimzi is first, which makes it easier to understand the sentence about what this document is about.

Reply:
And actually this makes me think about the sentence elsewhere about "The Strimzi project" making it easy to run Kafka on K8s/OS. I think we we need to make a clear distiniction about Strimzi-as-a-project and Strimzi as a "product" (by which I mean piece of software that's a product of the project). Really all this doc is about the software, so we should avoid talking about Strimzi as a "project" entirely.

Reply:
Ok, switched. @tombentley what about the other two places where @ppatierno suggests to use "... which ..."?

Reply:
Personally, I prefer two sentences in those cases too.

Reply:
English is really different from Italian ... two sentences in this way sounds really poor in Italian :-) Good to have you both guys better than me on that.

Reply:
@ppatierno Yeah, in Czech you would normally write whole paragraph in single sentence and it would look completely normal. So I can understand you. :-)

Reply:
> And actually this makes me think about the sentence elsewhere about "The Strimzi project" making it easy to run Kafka on K8s/OS. I think we we need to make a clear distiniction about Strimzi-as-a-project and Strimzi as a "product" (by which I mean piece of software that's a product of the project). Really all this doc is about the software, so we should avoid talking about Strimzi as a "project" entirely.

@tombentley I'm not sure I see the distinction here. But I can remove the work _project_ everywhere if that's what you meant.